components/camel-schematron/src/test/java/org/apache/camel/component/schematron/processor/SchematronProcessorTest.java

```diff
@@ -16,8 +16,8 @@
  */
 package org.apache.camel.component.schematron.processor;
 
-import javax.xml.transform.Templates;
-import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.*;
```

Comment:
W do not use * imports. Run: mvn compile -Psourcecheck to run checkstyle to see what problems you have